[Bulk] Broken Link fixes (#878)
MicrosoftDocs/biztalk-docs: biztalk/core/step-2-for-azure-create-an-edi-agreement.md

### To create partners

> 1.  Using your Microsoft account, log into the portal.

Reviewer: This doc is so outdated, I think when they say the "Azure BizTalk portal", they mean "Azure portal" and are referring to "Azure Logic Apps", which used to be called "BizTalk Services". Oh dear.

```suggestion
1. Sign in to the [Azure portal](https://portal.azure.com) with your Microsoft account.
```

> 2.  Create a partner for Northwind. Follow the steps at [Partners and Profiles](https://msdn.microsoft.com/library/windowsazure/hh689791) to create a partner.

Reviewer: ```suggestion
2. In the Azure portal search box, find and select **Integration accounts**. On the **Integration accounts** toolbar, select **Create**. Follow the prompts to create your integration account resource. 

3. After Azure creates the resource, open the integration account. On the resource menu, under **Settings**, select **Partners**. Create a partner for Northwind by following the steps to [Add trading partners to integration accounts](/azure/logic-apps/logic-apps-enterprise-integration-partners) to create a partner.
```
